src/client.jl

```diff
@@ -63,13 +64,19 @@ struct Client
     root::String
     headers::Dict{String,String}
     get_return_type::Function   # user provided hook to get return type from response data
-    clntoptions::Dict
-
-    function Client(root::String; headers::Dict{String,String}=Dict{String,String}(), get_return_type::Function=(default,data)->default, sslconfig=nothing, require_ssl_verification=true)
-        endswith(root, '/') && @warn("Root URI ($root) terminates with '/'. Ensure that resource paths do not begin with '/'. This is unconventional.")
-        clntoptions = Dict{Symbol,Any}(:status_exception=>false, :retries=>0, :require_ssl_verification=>require_ssl_verification)
-        (sslconfig === nothing) || (clntoptions[:sslconfig] = sslconfig)
-        new(root, headers, get_return_type, clntoptions)
```

Comment:
Is it ok to remove `sslconfig` here? what about the `require_ssl_verification` option?